tests/test_backend.py

```diff
@@ -2443,6 +2443,17 @@ def test_batch_to_spacend(self):
         _ = tf.batch_to_space_nd(input_x, block_size, crop, name=_TFOUTPUT)
         self._run_test_case([_OUTPUT], {_INPUT: input_val})
 
+    @check_opset_min_version(10, "Slice in opset 10 can accept dynamic 'starts' and 'ends'")
+    def test_batch_to_spacend_with_dynamic_crop(self):
+        block_size = [2, 2]
+        crops_val = np.array([[0, 1], [2, 1]], dtype=np.int32)
+        input_val = np.random.random_sample([40, 3, 5, 100]).astype(np.float32)
+
+        input_x = tf.placeholder(dtype=tf.float32, shape=input_val.shape, name=_TFINPUT)
+        crops = tf.placeholder(dtype=tf.int32, shape=[2, 2], name=_TFINPUT1)
```

Comment:
add test to cover int64

Reply:
Thanks! I have added a test to cover int64 type for the crops